bedrock/products/templates/products/vpn/platforms/windows.html

```diff
@@ -1,238 +1,36 @@
-    {% endcall %}
-
-    {% call vpn_content_media(
-    heading=ftl('vpn-windows-devices-headline'),
-    image=resp_img(
-      url='img/products/vpn/platforms/vpn-cntwell-devices.png',
-      srcset={
-        'img/products/vpn/platforms/vpn-cntwell-devices-high-res.png': '2x'
-      },
-      optional_attributes={
-        'width': '480',
-        'height': '480',
-        'loading': 'lazy'
-      }
-    ),
-    class_name='vpn-content-media-left-half'
-    ) %}
-    <p>{{ ftl('vpn-windows-devices-copy', devices=vpn_devices) }}</p>
-    {% endcall %}
-
-    {% call vpn_content_media(
-    heading=ftl('vpn-windows-speed-headline'),
-    image=resp_img(
-      url='img/products/vpn/platforms/vpn-cntwell-speed-guage.png',
-      srcset={
-        'img/products/vpn/platforms/vpn-cntwell-speed-guage-high-res.png': '2x'
-      },
-      optional_attributes={
-        'width': '480',
-        'height': '480',
-        'loading': 'lazy'
-      }
-    ),
-    class_name='vpn-content-media-right-half'
-    ) %}
-    <p>{{ ftl('vpn-windows-speed-copy') }}</p>
-    {% endcall %}
-
-    {% call vpn_content_media(
-    heading=ftl('vpn-windows-log-headline'),
-    image=resp_img(
-      url='img/products/vpn/platforms/vpn-cntwell-privacy.png',
-      srcset={
-        'img/products/vpn/platforms/vpn-cntwell-privacy-high-res.png': '2x'
-      },
-      optional_attributes={
-        'width': '480',
-        'height': '480',
-        'loading': 'lazy'
-      }
-    ),
-    class_name='vpn-content-media-left-half'
-    ) %}
-    <p>{{ ftl('vpn-windows-log-copy') }}</p>
-    {% endcall %}
-
-    {% call vpn_content_block(
-    class_name='t-highlight t-center'
-    ) %}
-    <h2 class="vpn-content-well-title">{{ ftl('vpn-shared-platform-cta-headline') }}</h2>
-    <div class="vpn-content-well-desc">
-      <a class="mzp-c-button mzp-t-xl" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-shared-platform-cta-button')
-        }}</a>
-      <p class="guarantee-copy">
-        <a href="{{ url('products.vpn.landing') }}#faq-refunds">{{ ftl('vpn-shared-money-back-guarantee') }}</a>
-      </p>
-    </div>
-    {% endcall %}
-
-  </div>
-
-  <footer class="vpn-footer mzp-l-content">
-    {% include 'products/vpn/includes/footer-legal.html' %}
-  </footer>
-
-</main>
+{% set subheading = self.page_title_full() %}
+{% set desc = ftl('vpn-windows-stay-safe') %}
+
+{% block article %}
+  <p>{{ ftl('vpn-windows-windows-is-one') }}</p>
+  <p>{{ ftl('vpn-windows-a-vpn-creates') }}</p>
+  <h3>{{ ftl('vpn-windows-what-are-the-benefits') }}</h3>
+  <ul class="mzp-u-list-styled">
+    <li>{{ ftl('vpn-windows-avoid-your-online') }}</li>
+    <li>{{ ftl('vpn-windows-if-youre') }}</li>
+    <li>{{ ftl('vpn-windows-you-can-exclude') }}</li>
+    <li>{{ ftl('vpn-windows-using-a-windows') }}</li>
+  </ul>
+  <p>{{ ftl('vpn-windows-the-mozilla') }}</p>
+
+  <h3>{{ ftl('vpn-windows-the-mozilla') }}</h3>
```

Comment:
**Question:** You're repeating the same string here. Maybe we should get rid of one?

Reply:
Lol this is currently on the page in production: https://www.mozilla.org/en-US/products/vpn/desktop/windows/

Good call-out, I'll remove the paragraph 👍